src/components/player/index.js

```diff
@@ -126,6 +85,38 @@ export default class Player extends React.Component {
     }
   }
 
+  handleKeyboardEvents = (e) => {
+    e.preventDefault();
```

Comment:
Typing in search would not be possible as you block all the default `keydown` events.

Reply:
Sorry, my bad, changing.